README.md

```diff
@@ -125,32 +125,11 @@ Please note that some toolboxes are for ArcGIS Pro only, or ArcGIS Desktop 10.3
 
 ### Downloading Test Data
 
-* Install and configure Apache Ant
-    * Download Ant from the [Apache Ant Project](http://ant.apache.org/bindownload.cgi) and unzip to a location on your machine, for example 'c:\apache-ant-1.9.2'.
-    * Set environment variable `ANT_HOME` to Ant Install Location (e.g. the folder from first step)
-    * Add Ant\bin to your path: `%ANT_HOME%\bin`
-    * NOTE: Ant requires Java [Runtime Environment (JRE) or Developer Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to be installed and the environment variable `JAVA_HOME` to be set to this location
-    * To verify your Ant Installation: Open Command Prompt> `ant -h` and verify it runs and returns the help options correctly 
-    * You may optionally install the [PyDev Eclipse Plugin for Python](http://pydev.org) if you plan to use Eclipse to run/debug
-* To download the data dependencies 
-    * Open Command Prompt>
-    * `cd solutions-geoprocessing-toolbox`
-    * `> ant`
-    * Verify “Build Succeeded”  
+**This section is under reconstruction, per issue [#371](https://github.com/Esri/solutions-geoprocessing-toolbox/issues/371)** 
 
 ### Running Verification Tests
 
-* Configure and verify Ant as described in the previous steps
-* To run all unit tests
-    * `> cd solutions-geoprocessing-toolbox`
-    * `>  ant -f run_all_tests.xml`
-    *  Note/Warning: this will run the test drivers from each test directory and can take several hours to run
-* To run individual unit tests
-    * Open Command Prompt>
-    * Go to the folder for the area you would like to test, the example below uses the `environment` area/folder, but each area has similar tests 
-    * `> cd solutions-geoprocessing-toolbox\environment\test`
-    * `> ant`
-    * Verify “Build Succeeded”
+**This section is under reconstruction, per issue [#371](https://github.com/Esri/solutions-geoprocessing-toolbox/issues/371)**
```

Comment:
Not sure what the schedule is for having a few basic tests running with downloadable test data, but if anyone besides @mfunk or @JudTown17 is going to test this we will at least need some steps for where/how to download the test data - so that should probably be added back before release (even if manual install steps).